### DISCUSSION TITLE
Preserving tabs in docx

### DISCUSSION THREAD

**Discussion by UK-Peter** [jgm/pandoc#10276] [Q&A]
Hi, I need all tab characters preserved when converting a docx to plain text. I've tried --preserve-tabs=true but for some reason this only preserves tabs in code blocks. I want ALL tabs preserved, it's vital for the project I'm working on. Thanks. 
Peter

**Comment by jgm** [comment]
Can you explain more about why?  Most of the formats pandoc supports treat strings of spaces and tabs interchangeably, so we don't distinguish between them (except in code contexts, where it can matter).  This is certainly true of markdown, and the plain text writer is a dumbing-down of the markdown writer.

**Comment by UK-Peter** [comment]
Thanks for the reply. I have Word documents of auction listings (used for a printed catalog) with lot number, description and price separated by a tab. I want to extract the plain text, then another piece of software parses that and creates an HTML page. So it would be super useful to preserve all tabs. But there's workarounds of course, it's not a big problem.

**Reply by jgm** [reply]
One solution I can think of:  preprocess the docx, converting all tabs inside document.xml to some other character sequence ("XXXXX"), then this will be preserved by pandoc and you can process it later.  This might still be easier than processing the XML directly.
